Misc/NEWS.d/next/Core_and_Builtins/2024-08-12-11-45-47.gh-issue-122821.WnAzTK.rst

```diff
@@ -0,0 +1,3 @@
+Make sure that branches in while statements have consistent offsets for
+sys.monitoring. ``while`` statements are now compiled with a simple jump at
+the end of the body, instead of duplicating the test.
```

Comment:
```suggestion
Make sure that branches in :keyword:`while` statements have consistent offsets for
:mod:`sys.monitoring`. :keyword:`!while` statements are now compiled with a simple
jump at the end of the body, instead of duplicating the test.
```